docs/user/alerting/alert-types.asciidoc

```diff
@@ -1,159 +1,28 @@
 [role="xpack"]
 [[alert-types]]
-== Standard stack alert types
+== Alert types
 
-{kib} supplies alert types in two ways: some are built into {kib} (these are known as stack alerts), while domain-specific alert types are registered by {kib} apps such as <<xpack-apm,*APM*>>, <<xpack-ml,*{ml-cap}*>>, <<metrics-app,*Metrics*>>, and <<uptime-app,*Uptime*>>.
+{kib} supplies alert types in two ways: some are built into {kib} (these are known as stack alerts), while domain-specific alert types are registered by {kib} apps.
 
-This section covers stack alerts. For domain-specific alert types, refer to the documentation for that app.
+[float]
+==== Standard stack alert types
+
+This section covers stack alerts.
 Users will need `all` access to the *Stack Alerts* feature to be able to create and edit any of the alerts listed below.
 See <<kibana-feature-privileges, feature privileges>> for more information on configuring roles that provide access to this feature. 
 
 Currently {kib} provides two stack alerts: <<alert-type-index-threshold>> and <<alert-type-es-query>>.
```

Comment:
```suggestion
{kib} provides two stack alerts: 

* <<alert-type-index-threshold>>
* <<alert-type-es-query>>

```

Reply:
Should the geo alert be added to this list?

Reply:
No, I spoke with Maps team and they want it to be separately, because they are planning more alerts types for Maps.

Reply:
Yeah, this was awkward when I was doing the docs from the geo side too. On one hand, Geo Containment is a stack alert and I expect the next one, Geo Proximity, will be one too. It's possible we'll do more beyond this but that's TBD. They're distinct from other stack alerts in that they're geo-specific and require a Gold+ license, however I'm aware that's not something we need to distinguish in the docs. I guess I'm flexible in where and how we list them. @gchaps you probably have the best bird's eye view on docs. Any recommendations here?

Reply:
Let's get this PR out and then review the placement of the geo alert at a later date.